source/change-streams.rst

```diff
@@ -41,7 +41,62 @@ Terms
 Resumable Error
 ^^^^^^^^^^^^^^^
 
-Any error encountered which is not a server error, with the exception of server responses with the message “not master” or error code 43 (cursor not found).  An example might be a timeout error, or network error.
+An error is considered resumable if it meets any of the following criteria:
+
+- any error encountered which is not a server error (e.g. a timeout error or
+  network error)
+
+- a server error response with any of the following error codes:
+
+  .. list-table::
+    :header-rows: 1
+
+    * - Error Name
+      - Error Code
+    * - HostUnreachable
+      - 6
+    * - HostNotFound
+      - 7
+    * - NetworkTimeout
+      - 89
+    * - SocketException
+      - 9001
+    * - NotMaster
+      - 10107
+    * - NotMasterNoSlaveOk
+      - 13435
+    * - NotMasterOrSecondary
+      - 13436
+    * - InterruptedDueToReplStateChange
+      - 11602
+    * - PrimarySteppedDown
+      - 189
+    * - ExceededTimeLimit
+      - 50
+    * - RetryChangeStream
```

Comment:
Maybe also
- 'ElectionInProgress': https://github.com/mongodb/mongo/blob/0c1402ee3f10eaf9d6d989d814bc3113a380091e/src/mongo/base/error_codes.err#L216

Reply:
Can "ElectionInProgress" be returned from any command? Should this also be an SDAM "not master" error?

Reply:
Response from Andy:

> [ElectionInProgress] shouldn't happen in response to CRUD commands. It happens when a node tries to switch from SECONDARY to another non-PRIMARY mode (like ROLLBACK) while simultaneously running for election to primary. The code shouldn't really ever reach the network.

So it does not need to be in SDAM's list of error codes.